app/assets/javascripts/components/bulkUploadImport.jsx

```diff
@@ -272,7 +300,8 @@ class BulkUploadImport extends React.Component {
     const samples = this.state.samples
     samples[e.target.id].project_id = this.state.allProjects[e.target.selectedIndex].id
     this.setState({
-      samples: samples
+      samples: samples,
+      project: e.target.value.trim()
```

Comment:
what is this line for? why change project when you just change project for one sample.